arithmetization/src/main/java/net/consensys/linea/zktracer/module/ecdata/EcData.java

```diff
@@ -37,14 +37,15 @@
 @RequiredArgsConstructor
 public class EcData implements Module {
   public static final Set<Address> EC_PRECOMPILES =
-      Set.of(Address.ECREC); // TODO: add later Address.ALTBN128_ADD, Address.ALTBN128_MUL,
-  // Address.ALTBN128_PAIRING);
+      Set.of(Address.ECREC, Address.ALTBN128_ADD, Address.ALTBN128_MUL, Address.ALTBN128_PAIRING);
 
   @Getter private final StackedSet<EcDataOperation> operations = new StackedSet<>();
   private final Hub hub;
   private final Wcp wcp;
   private final Ext ext;
 
+  @Getter private EcDataOperation ecdDataOperation;
```

Comment:
could be final

Reply:
If you guarantee an operation is created (see return cases below) yes. The triggering approach is WIP everywhere, but I guess it will be the case.